deploy/resources/maistra/maistra-controlplane-0.10.0.yaml

```diff
@@ -20,6 +20,22 @@ spec:
         autoscaleEnabled: false
       istio-egressgateway:
         enabled: false
+      cluster-local-gateway:
+        autoscaleEnabled: false
+        enabled: true
+        labels:
+          app: cluster-local-gateway
+          istio: cluster-local-gateway
+          chart: gateways
+          heritage: Tiller
+          maistra-version: 0.10.0
+          release: istio
```

Comment:
Are all of these labels necessary?

Seems like upstream only sets the first two: https://github.com/knative/serving/blob/master/third_party/istio-1.2.0/values.yaml#L28-L30

Reply:
Yes it will work with first 2 labels.
Updated now.